wordpress-mobile/WordPress-iOS
# Update GB reference to 1.33.0 hash

Integrate version 1.33.0 of Gutenberg for the block editor.

GB PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2493

To test:
 - Smoke test the main application flows
 - Check the following new features:
   - [***] Media editing support in Media & Text block.
   - [***] New block: Social Icons
   - [*] Cover block placeholder is updated to allow users to start the block with a background color

PR submission checklist:
